youtube_dl/extractor/adobepass.py

```diff
@@ -325,6 +320,11 @@
     'col080': {
         'name': 'Columbus Telephone'
     },
+    'Comcast_SSO': {
+        'name': 'Comcast XFINITY',
+        'username_field': 'user',
+        'password_field': 'passwd',
+    },
```

Comment:
Oh, OK.  Then I'll revise the MSO list to leave the grouping as is.  Are we grouping the new manually-added MSOs in any particular order?